ui-tests/tests/search.spec.ts

```diff
@@ -1,25 +1,66 @@
-import { test } from '@jupyterlab/galata';
+import { test, galata } from '@jupyterlab/galata';
 import { expect } from '@playwright/test';
+import * as path from 'path';
 
-test('test', async ({ page }) => {
+const fileName = 'conftest.py';
+test.use({ tmpPath: 'search-replace-test' });
+
+test.beforeAll(async ({ baseURL, tmpPath }) => {
+  const contents = galata.newContentsHelper(baseURL);
+  await contents.uploadFile(
+    path.resolve(
+      __dirname,
+      `../../jupyterlab_search_replace/tests/${fileName}`
+    ),
+    `${tmpPath}/${fileName}`
+  );
+});
+
+test.afterAll(async ({ baseURL, tmpPath }) => {
+  const contents = galata.newContentsHelper(baseURL);
+  await contents.deleteDirectory(tmpPath);
+});
+
+test('should get 5 matches', async ({ page }) => {
```

Comment:
As you included #22 in this one, could you add a test for opening a file from a click.

Reply:
done